projects/unore/index.js

```diff
@@ -25,6 +26,18 @@ const ethRiskPools = [
 ]
 const ethSSIPEth = '0x29B4b8674D93b36Bf651d0b86A8e5bE3c378aCF4'
 
+// KAVA pools
+// tokens
+const kavaTokens = [
+  // '0xfA9343C3897324496A05fC75abeD6bAC29f8A40f', // USDC
```

Comment:
shouldn't this list be enabled?

Reply:
This is USDC address which KAVA team provided us, but this USDC coin is not recognized when I tried to calculate tvl,
So I would like to keep it now if possible, so we can remember and update without missing a bit later.

Reply:
just confirming, I can fix it